Dockerfile

```
@@ -1,4 +1,5 @@
-FROM python:3.7
+ARG BASE=3.8
```

```suggestion
ARG BASE=3.8-slim
```

Might try using the `slim` image to bring down the size?